src/main/resources/assets/patchouli/lang/de_DE.lang

```diff
@@ -0,0 +1,93 @@
+# Item Names
+item.patchouli:guide_book.name=Anleitungsbuch
```

Comment:
Well this book might contain anything, that's the issue here; this sounds too much like an instruction manual. I am still not happy with the name...

Reply:
Is "Handbuch" an option? @Vexatos